packages/closer/pages/learn/category/[slug]/index.tsx

```
@@ -17,6 +17,7 @@ import { useTranslations } from 'next-intl';
 import { useAuth } from '../../../../contexts/auth';
```

ESLint

[error] 'useAuth' is defined but never used.

(unused-imports/no-unused-imports)